microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -85,10 +85,26 @@ After you launch Microsoft Edge \(Chromium\) using a command-line argument, IE m
 
 IE mode remains unavailable until you completely close and restart Microsoft Edge \(Chromium\).  
 
+
+## Automating IE mode
+
+If you have an existing application that uses the [InternetExplorer][InternetExplorerObject] object to automate Internet Explorer 11, your application will not work if the Internet Explorer 11 desktop application is not available.  Microsoft Edge \(Chromium\) does not support automating IE mode through the InternetExplorer object, so applications that rely on this object will need to be updated using one of the recommended alternatives below.  Applications that use the [WebBrowser][WebBrowserControl] control will continue to work and will not be affected by the removal of the Internet Explorer 11.
```

Comment:
The final "the" of the paragraph should either be removed or "Desktop Application" should follow the 11.

Reply:
We can also probably drop all of the **(Chromium)** references in this section. (Soon we should remove them from the whole doc.)

Reply:
Fixed the typo. I'm leaving the (Chromium) references as-is for now since that's currently used throughout the documentation.